src/main/java/selenium/Driver.java

```diff
@@ -0,0 +1,22 @@
+package selenium;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Driver {
+    private static WebDriver driver;
+
+    private Driver(){}
+
+    public static WebDriver getInstance() {
+        if (driver == null) {
+            driver = new ChromeDriver();
+        }
+        return driver;
+    }
+
+    public static void close() {
+        driver.quit();
+        driver = null;
+    }
+}
```

Comment:
Notes: access to WebDriver instance must be done via singleton pattern. I don't see a singleton here

Reply:
@KBugrimCS  It is a singleton created in accordance with the following article: https://refactoring.guru/design-patterns/singleton/java/example#example-0. Please, provide more details on what kind of implementation you want to see.

Reply:
No, it's not a Singleton in your case. I do not know what kind of information you need to provide, if you threw the link to the Singleton pattern, I only ask you to implement it correctly

Reply:
Updated